assets/dist/rolodex.js

```diff
@@ -1,5 +1,5 @@
 (function() {
-  angular.module('rolodex', ['templates', 'rolodex.accordion', 'rolodex.alert', 'rolodex.buttons', 'rolodex.datepicker', 'rolodex.dropdown', 'rolodex.modal', 'rolodex.transition']);
```

Comment:
it was incorrectly assuming `templates` existed. that came from the `angular-html2js` gem. i had a gulp task to add an empty module definition to the compiled JS for our SPA apps. instead, it just uses the module namespace